Python/pywarpx/picmi.py

```diff
@@ -579,17 +581,17 @@ def initialize_inputs(self):
 class ElectrostaticSolver(picmistandard.PICMI_ElectrostaticSolver):
     def init(self, kw):
         self.relativistic = kw.pop('warpx_relativistic', False)
-
+        self.self_fields_verbosity = kw.pop('warpx_self_fields_verbosity', None)
     def initialize_inputs(self):
 
         self.grid.initialize_inputs()
-
```

Comment:
This is ok, but it's better to avoid making formatting changes unrelated to the PR changes.